app/views/placements/schools/placements/edit_provider.html.erb

```diff
@@ -6,24 +6,34 @@
 <% end %>
 
 <div class="govuk-width-container">
-  <%= render AutocompleteSelectFormComponent.new(
-    model: @placement,
-    scope: :placement,
-    url: placements_school_placement_path(@school, @placement),
-    method: :patch,
-    data: {
-      autocomplete_path_value: "/api/provider_suggestions",
-      autocomplete_return_attributes_value: %w[code],
-      input_name: "placement[provider_name]",
-    },
-    input: {
-      field_name: :provider_id,
-      value: @placement.provider_name,
-      label: t(".title"),
-      caption: "Edit Placement",
-      previous_search: @placement.provider_id,
-    },
-  ) %>
+  <%= form_for(@placement, url: placements_school_placement_path(@school, @placement), method: :put) do |f| %>
+    <%= f.govuk_error_summary %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-l"><%= t(".caption") %></span>
+
+        <%= f.govuk_radio_buttons_fieldset :subject_ids, legend: { size: "l", text: t(".provider") } do %>
```

Comment:
It looks like this isn't breaking any functionality, but it'd still be good to tidy it up.

```suggestion
        <%= f.govuk_radio_buttons_fieldset :provider_id, legend: { size: "l", text: t(".provider") } do %>
```